packages/arb-token-bridge-ui/src/components/common/Dialog2.tsx

```diff
@@ -71,6 +82,12 @@ export function DialogWrapper(props: DialogProps) {
   const isOftTransfer = useIsOftV2Transfer()
   const [selectedToken] = useSelectedToken()
   const [{ amount }] = useArbQueryParams()
+  const [networks] = useNetworks()
+  const latestNetworks = useLatest(networks)
+  const {
+    current: { childChainProvider }
+  } = useLatest(useNetworksRelationship(latestNetworks.current))
+  const nativeCurrency = useNativeCurrency({ provider: childChainProvider })
```

Comment:
https://github.com/streamich/react-use/blob/master/docs/useLatest.md

`useLatest` should only be needed for using the values in async functions
just looking at the code it shouldn’t be a problem
can you please check if it hit an issue?
we want to try to not use `useLatest` whenever possible